physionet-django/project/utility.py

```diff
@@ -677,3 +679,42 @@ def convert_url(self, attr_name, url):
             if newurl != url:
                 return newurl
         return url
+
+
+class DataSourceCreator:
+    def __init__(self, **kwargs):
+        self.data_location = kwargs.get('data_location', None)
+        self.files_available = kwargs.get('files_available', None)
+        self.email = kwargs.get('email', None)
+        self.uri = kwargs.get('uri', None)
+        self.access_mechanism = kwargs.get('access_mechanism', None)
```

Comment:
Hmm, why did you opt for keyword arguments here? Why not:
```suggestion
    def __init__(self, data_location=None, files_available=None, email=None, uri=None, access_mechanism=None):
        self.data_location = data_location
        self.files_available = files_available
        self.email = email
        self.uri = uri
        self.access_mechanism = access_mechanism
```